ring_buffer/ring_buffer.py

```diff
@@ -5,7 +5,11 @@ def __init__(self, capacity):
     self.storage = [None]*capacity
 
   def append(self, item):
-    pass
+    self.storage[self.current] = item
+    if self.current == len(self.storage) - 1:
+      self.current = 0
+    else:
+      self.current += 1
 
   def get(self):
-    pass
\ No newline at end of file
+    return filter(lambda x: x is not None, self.storage)
```

Comment:
I'm getting a failed test on line #16 from `test_ring_buffer.py` Kyle. But it looks like it's just that the return value `filter object` hasn't been fully unpacked. Code is returning `<filter object at 0x10ac76eb8>`and test expects `['a','b','c','d']`.